Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -10,6 +10,8 @@ class OWDistanceTransformation(widget.OWWidget):
     description = "Transform distances according to selected criteria."
     icon = "icons/DistancesTransformation.svg"
 
+    learner_name = settings.Setting("Distance Transformation")
+
```

Comment:
Nope, the heading comes from [here](https://github.com/biolab/orange3/pull/1113#discussion-diff-56058820R9). Try it.